app/controllers/observations_controller/index.rb

```diff
@@ -28,12 +28,6 @@ def list_observations
     show_selected_observations(query)
   end
 
-  # Displays matrix of all Observations, alphabetically.
-  def observations_by_name
-    query = create_query(:Observation, :all, by: :name)
-    show_selected_observations(query)
-  end
```

Comment:
Removes unreferenced code.
@pellaea, @nimmolo: Am I overlooking some use of this code e.g., in meta-programming?

Reply:
Naw, I never used this even when there was still a way to get to this!